python3-aeneas.yml

```diff
@@ -0,0 +1,99 @@
+# NOTES:
+  # 1.7.3.0 is the latest aeneas version on pypi. Are there newer versions
+  # and sources elsewhere? 
+  # This file uses version 1.25.2 of numpy which worked without having to
+  # install extra build modules. The latest version of numpy is 2.0.0 which
+  # will require 'meson-python' to build successfully.
+  # A setup error in version 1.17.5 of numpy suggested the "setup.py clean"
+  # function was deprecated for builds. This is why I was testing new versions.
+  # FYI: The last version of numpy released as a .zip instead of a .tar.gz
+  # was 1.22.4: https://pypi.org/project/numpy/1.22.4/#files
+name: python3-aeneas
+buildsystem: simple
+build-options:
+    append-ld-library-path: /app/lib
+build-commands:
+    - echo PWD="${PWD}"  and  FLATPAK_DEST="${FLATPAK_DEST}"
```

Comment:
```suggestion
```

Reply:
What is different? Replace all 4 lines with just the 1 line?

This file comes from a app already accepted into flathub:

https://github.com/flathub/org.sil.Bloom/blob/master/python3-aeneas.yml

Reply:
$PWD is a shell built in and $FLATPAK_DEST is already set. The source you are pointing at is doing it wrong.

echo -ing this is not a necessary part of the build process

Reply:
Removed